plugins/module_utils/network/sonic/config/bgp_neighbors/bgp_neighbors.py

```diff
@@ -484,6 +533,36 @@ def build_bgp_peer_groups_payload(self, cmd, have, bgp_as, vrf_name):
         payload = {'openconfig-network-instance:peer-groups': {'peer-group': bgp_peer_group_list}}
         return payload, requests
 
+    def get_afi_safi_config_payload(self, ip_afi):
```

Comment:
Please rename this function to reduce confusion with the higher (generic) level afi_safi category. It would be good to explicitly include "ip" in the name to clarify that fact that it is to be used only for the IP specific afi_safi cases.

e.g. get_ip_afi_config_payload or get_ip_afi_safi_config_payload